tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_CONUScompact_25km_ics_RRFS_lbcs_RRFS_suite_RRFS_sas.yaml

```
@@ -0,0 +1,29 @@
+metadata:
+  description: |-
+    This test is to ensure that the workflow running in community mode
+    completes successfully on the RRFS_CONUScompact_25km grid using the RRFS_v1beta
+    physics suite. It uses RRFS forecasts mapped onto 3-km regular grid (rrfs*.conus.grib2) for
+    ICs and LBCs. This test uses old v1 sfc_data.nc format, not the v2 sfc_data.
```

There is no reason to include `v1/v2 sfc_data` in the description, since `RRFS_v1` doesn't use `v2 sfc_data`.

```suggestion
    ICs and LBCs.
```

---

Updated in the recent commits